### ISSUE TITLE
UnboundLocalError

### ISSUE THREAD

**Issue by Jeor** [aeon0/d4lf#25]
Getting the following error:

Exception in thread Thread-7 (_wrapper_run_loot_filter):
Traceback (most recent call last):
  File "threading.py", line 1045, in _bootstrap_inner
  File "threading.py", line 982, in run
  File "overlay.py", line 139, in _wrapper_run_loot_filter
  File "loot_filter.py", line 86, in run_loot_filter
  File "loot_filter.py", line 65, in check_items
  File "item\read_descr.py", line 205, in read_descr
UnboundLocalError: cannot access local variable 'lines_to_add' where it is not associated with a value

It happens when scanning this item specifically: 
![Picture1](https://github.com/aeon0/d4lf/assets/4705491/3f87d095-2892-45bb-a714-516b9a88de69)

aspects & affixes below for reference.
```
# Necromancer
- [aspect_of_ultimate_shadow, 2900]
- [bloodsoaked_aspect, 1800]
- [aspect_of_grasping_veins]
- [blighted_aspect, 100]
- [aspect_of_shielding_storm, 5]
- [conceited_aspect, 25]
- [aspect_of_exposed_flesh]
- [splintering_aspect]
- [aspect_of_serration]
```

```
- Armor:
    itemType: [chest armor, pants]
    minPower: 750
    affixPool:
    - [damage_reduction_from_close_enemies, 10]
    - [damage_reduction_from_affected_by_shadow_damage_over_time_enemies, 3.5]
    - [damage_reduction, 10]
    - [maximum_life, 900]
    minAffixCount: 3
```

**Comment by aeon0** [comment]
Thanks for the issue. This actually has two issues:
1) The name "shield" is used to wrongly classify the item as shield instead of pants. Will make sure the later will be used.
2) It seems the templates I got from someone else for the 1440p are done with font size medium instead of small... I created some properl templates now (with DSR upscaling)

**Comment by aeon0** [comment]
fixed on latest main and in upcoming release 1.3.3
